test-harness/README.md

```diff
@@ -36,7 +36,7 @@ Test text, can be multi line.
 ====spec====
 openapi 2/3 document
 ====server====
-command line arguments to run Prism with
+command line arguments to run Prism with. You can use the ${file} template variable
```

Comment:
@XVincentX, you `can` or `have to`? Also, don't we always mean `document` or `spec` by `file`? Maybe it's worth renaming? WDYT?

Reply:
You're correct, it's compulsory.

Phil has always been against using `spec` — so definitely not that term. Let's leave it as it is right now (otherwise merging this in the Proxy PR is going to be a mess) and we can change the name if we find something appropriate.

Reply:
The process times out for me, when I remove ${file} from the tests

Reply:
Docs rewrote.